src/components/Card.js

```diff
@@ -26,12 +39,19 @@ export default class Card {
   }
 
   /**Event Handlers */
-  _handleLike() {
+  handleLike() {
     this._cardElement
       .querySelector(".gallery__card-button")
```

Comment:
You've found the like button in line 69 and it's accessible here as `this._likeButton`


```js
  handleLike() {
    this._likeButton.classList.toggle("gallery__card-button_active");
  }

```